src/main/java/ganymedes01/etfuturum/api/DeepslateOreRegistry.java

```diff
@@ -57,7 +57,7 @@ public static void addOre(Block from, Block to) {
 	 * @param toMeta   The meta deepslate changes it to
 	 */
 	public static void addOre(Block from, int fromMeta, Block to, int toMeta) {
-		if (from instanceof ITileEntityProvider || to instanceof ITileEntityProvider) {
+		if (from.hasTileEntity(fromMeta) || to.hasTileEntity(toMeta)) {
```

Comment:
Revert this; I will add it later but I need to test this to ensure it does not have any additional undesired side-effects

Reply:
Yes Sir, it's reverted. 😊